### PR TITLE
[BUGFIX] Fix glue test result serialization

### PR DESCRIPTION
**Pull request by eric-haibin-lin**
## Description ##
Make test result.tsv consistent with the format required on GLUEBenchmark. 

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by mli** [comment]
Job PR-773/1 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-773/1/index.html

File: scripts/bert/finetune_classifier.py [hunk 1]
```diff
@@ -336,26 +336,33 @@ def test(loader_test, segment):
...
-        values, indices = mx.nd.topk(out, k=1, ret_typ='both')
-        value_list.extend(values.asnumpy().reshape(-1).tolist())
-        index_list.extend(indices.asnumpy().reshape(-1).tolist())
+        if not task.class_labels:
+            # regression task
+            for result in out.asnumpy().reshape(-1).tolist():
+                results.append('{:.3f}'.format(result))
+        else:
+            # classification task
+            indices = mx.nd.topk(out, k=1, ret_typ='indices').asnumpy()
```
Comment (szha) [inline scripts/bert/finetune_classifier.py]: dtype='int32'?

Comment (szha) [reply scripts/bert/finetune_classifier.py]:
```
In [1]: import mxnet as mx

In [2]: a = mx.random.uniform(shape=(16777217,))

In [3]: b = a.topk(k=16777217, ret_typ='indices')

In [4]: (a.dtype, b.dtype)
Out[4]: (numpy.float32, numpy.float32)

In [5]: b
Out[5]: ---------------------------------------------------------------------------
MXNetError                                Traceback (most recent call last)
/usr/local/Cellar/ipython/7.5.0/libexec/lib/python3.7/site-packages/IPython/core/formatters.py in __call__(self, obj)
    700                 type_pprinters=self.type_printers,
    701                 deferred_pprinters=self.deferred_printers)
--> 702             printer.pretty(obj)
    703             printer.flush()
    704             return stream.getvalue()

/usr/local/Cellar/ipython/7.5.0/libexec/lib/python3.7/site-packages/IPython/lib/pretty.py in pretty(self, obj)
    400                         if cls is not object \
    401                                 and callable(cls.__dict__.get('__repr__')):
--> 402                             return _repr_pprint(obj, self, cycle)
    403
    404             return _default_pprint(obj, self, cycle)

/usr/local/Cellar/ipython/7.5.0/libexec/lib/python3.7/site-packages/IPython/lib/pretty.py in _repr_pprint(obj, p, cycle)
    695     """A pprint that just redirects to the normal repr function."""
    696     # Find newlines and replace them with p.break_()
--> 697     output = repr(obj)
    698     for idx,output_line in enumerate(output.splitlines()):
    699         if idx:

/usr/local/lib/python3.7/site-packages/mxnet/ndarray/ndarray.py in __repr__(self)
    192         """Returns a string representation of the array."""
    193         shape_info = 'x'.join(['%d' % x for x in self.shape])
--> 194         return '\n%s\n<%s %s @%s>' % (str(self.asnumpy()),
    195                                       self.__class__.__name__,
    196                                       shape_info, self.context)

/usr/local/lib/python3.7/site-packages/mxnet/ndarray/ndarray.py in asnumpy(self)
   1994             self.handle,
   1995             data.ctypes.data_as(ctypes.c_void_p),
-> 1996             ctypes.c_size_t(data.size)))
   1997         return data
   1998

/usr/local/lib/python3.7/site-packages/mxnet/base.py in check_call(ret)
    251     """
    252     if ret != 0:
--> 253         raise MXNetError(py_str(_LIB.MXGetLastError()))
    254
    255

MXNetError: [16:45:37] src/operator/tensor/./ordering_op-inl.h:393: Check failed: element_num <= mxnet::common::MaxIntegerValue<IDType>() (16777217 vs. 16777216) : 'IDType' does not have a sufficient precision to represent the indices of the input array. The total element_num is 16777217, but the selected IDType can only represent 16777216 elements
```

**Comment by mli** [comment]
Job PR-773/2 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-773/2/index.html

**Pull request merged**